qml/MKSConnectBtn.qml

```diff
@@ -61,7 +61,7 @@ Item {
             onClicked: Cura.MachineManager.printerOutputDevices[0].requestAuthentication();
             // style: UM.Theme.styles.print_setup_action_button;
             text: catalog.i18nc("@action:button", "Request Access");
-            tooltip: catalog.i18nc("@info:tooltip", "Send access request to the printer");
+            // tooltip: catalog.i18nc("@info:tooltip", "Send access request to the printer");
```

Comment:
This is something I'm still trying to figure out. I know that attempting to use the `tooltip` property for Qt's `Button` will cause an error, as it's been removed. It might still be in Cura.PrimaryButton or another ButtonType? If not, I think it can be added via a onHover method or something. I've just commented them out for now so the plugin will load in Cura 5.0, but it certainly should be addressed

Reply:
Found it. https://doc-snapshots.qt.io/qt6-dev/qml-qtquick-controls2-tooltip.html#details

I will update all tooltips accordingly :)